package.json

```diff
@@ -10,15 +10,22 @@
   },
   "author": "Event Espresso",
   "license": "GPL-2.0",
+  "browserslist": [
+    "extends @wordpress/browserslist-config"
+  ],
```

Comment:
This is super nifty because there's a number of packages that support the `browserslist` configuration.  I think one of the `babel` plugins we are using also follows it.

Reply:
ya this was a great idea to add this.
also looking forward to not having to add vendor prefixes to css due to autoprefixer.